packages/editor-ui/src/Interface.ts

```diff
@@ -314,8 +314,29 @@ export interface IWorkflowDb {
 	versionId: string;
 	usedCredentials?: IUsedCredential[];
 	meta?: WorkflowMetadata;
+	resource?: 'workflow';
 }
 
+// For workflow list we don't need the full workflow data
+export type WorkflowResourceDB = Omit<
+	IWorkflowDb,
+	'nodes' | 'connections' | 'settings' | 'pinData' | 'versionId' | 'usedCredentials' | 'meta'
+>;
+export interface IFolderDb {
```

Comment:
I think we have been dropping the `I` for `Interface` in our types. This can be just `Folder` or `FolderResource`